### PR TITLE
fix: forward blockchain path for simple redirect

### PR DESCRIPTION
**Pull request by crisog**
This allows blockchains with a path, to be used with DNS redirects.

### CODE REVIEW

File: src/controllers/v1.controller.ts [hunk 1]
```diff
@@ -144,7 +144,7 @@ export class V1Controller {
       // TODO: Move URL to ENV
       this.host = `${blockchainAliases[0]}.gateway.pokt.network`
 
-      const { blockchainRedirects } = await loadBlockchain(
+      const { blockchainRedirects, blockchainPath } = await loadBlockchain(
```
Comment (nymd) [inline src/controllers/v1.controller.ts:147]: This should get a change in the blockchains.model too yea?

**Comment by crisog** [comment]
Close T-2716

File: src/controllers/v1.controller.ts (hunk 1 above)
Comment (crisog) [reply src/controllers/v1.controller.ts:147]: We moved the path to sync check, but it's no longer useful ONLY for the sync checks. We should go back to using the field outside of `syncCheckOptions`

Comment (crisog) [reply src/controllers/v1.controller.ts:147]: I added `path` field to all of the blockchains that had a path in the DB already.

Comment (crisog) [reply src/controllers/v1.controller.ts:147]: > This should get a change in the blockchains.model too yea?

Also, the `path` field was never removed from the model so no need to change.

**Review by nymd — APPROVED** [review]

**Pull request merged**